src/core/logging.py

```
@@ -0,0 +1,73 @@
+import logging
+import sys
+from typing import Any, Optional
+
+import structlog
+from structlog.types import EventDict, Processor
+
+
+def add_timestamp(logger: Any, method_name: str, event_dict: EventDict) -> EventDict:
+    """タイムスタンプをISO形式で追加"""
+    from datetime import datetime, timezone
+    event_dict["timestamp"] = datetime.now(timezone.utc).isoformat().replace("+00:00", "Z")
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)